### PR TITLE
Changes on matplotlib notebook

### PR DESCRIPTION
**Pull request by jpherdi**
I changed a little on the matplotlib notebook. Please check.

### CODE REVIEW

File: python/matplotlib.ipynb [hunk 1]
```diff
@@ -1,748 +1,730 @@
...
+            "outputs": [],
+            "source": [
+                "plt.plot(t, np.sin(t), 'go')"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "Tabelle mit allen Farben und Styles: [matplotlib.axes.Axes.plot](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.plot.html)\n",
```
Comment (chrbeckm) [inline python/matplotlib.ipynb:118]: Stilen

File: python/matplotlib.ipynb [hunk 2]
```diff
@@ -1,748 +1,730 @@
...
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "plt.plot(t, np.sin(t))\n",
+                "plt.xlabel(r'$t / \\mathrm{s}$')\n",
+                "plt.ylabel(r'$U \\,/\\, \\mathrm{V}$')  # Spaces sind Geschmacksfrage……"
+            ]
```
Comment (chrbeckm) [inline python/matplotlib.ipynb:199]: Collapse into one cell?

File: python/matplotlib.ipynb [hunk 3]
```diff
@@ -1,748 +1,730 @@
...
+                "# Anzahl Zeile, Anzahl Spalten, Nummer des Plots\n",
+                "plt.subplot(2, 1, 1)\n",
+                "plt.plot(x, x**2)\n",
+                "plt.xlim(0, 2 * np.pi)\n",
+                "plt.title(r\"$f(x)=x^2$\")\n",
+                "\n",
+                "plt.subplot(2, 1, 2)\n",
+                "plt.plot(x, np.sin(x))\n",
+                "plt.xlim(0, 2 * np.pi)\n",
+                "plt.title(r\"$f(x)=\\sin(x)$\")"
```
Comment (chrbeckm) [inline python/matplotlib.ipynb:457]: Didn't work for me in Google Colab, will check later and proper.

File: python/matplotlib.ipynb [hunk 4]
```diff
@@ -1,748 +1,730 @@
...
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "# Zufallsdaten generieren:\n",
+                "x = np.random.normal(0, 1, 1000)\n",
+                "plt.hist(x, bins=20);\n",
```
Comment (chrbeckm) [inline python/matplotlib.ipynb:619]: I understand the `;`, maybe explain at the start and use it with every plot?

File: python/matplotlib.ipynb [hunk 5]
```diff
@@ -1,748 +1,730 @@
...
+                "z = np.linspace(-2, 2, 200)\n",
+                "\n",
+                "r = z**2 + 1\n",
+                "x = r * np.sin(theta)\n",
+                "y = r * np.cos(theta)\n",
+                "\n",
+                "ax.plot(x, y, z, label='parametric curve')\n",
+                "ax.legend()\n",
+                "\n",
+                "plt.show()"
```
Comment (chrbeckm) [inline python/matplotlib.ipynb:687]: not needed

File: python/matplotlib.ipynb [hunk 6]
```diff
@@ -1,748 +1,730 @@
...
+                "\n",
+                "https://matplotlib.org/mpl_toolkits/mplot3d/tutorial.html"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "<h1> LaTeX </h1>\n",
+                "<b1> Es ist auch möglich LaTeX für das Setzen aller Plot-Beschriftungen (d.h. Achsenbeschriftungen, Ticks, Legenden, usw.) zu verwenden. Seht Euch dazu die \"TeX in matplotlib\" Folien an. </b1>"
```
Comment (chrbeckm) [inline python/matplotlib.ipynb:704]: Convert from HTML to md

File: python/matplotlib.ipynb (hunk 3 above)
Comment (chrbeckm) [reply python/matplotlib.ipynb:457]: Did work in jupyter

File: python/matplotlib.ipynb (hunk 2 above)
Comment (jpherdi) [reply python/matplotlib.ipynb]: Is the way alright, in which I did it now?

File: python/matplotlib.ipynb (hunk 4 above)
Comment (jpherdi) [reply python/matplotlib.ipynb]: yes, good idea.

File: python/matplotlib.ipynb (hunk 5 above)
Comment (jpherdi) [reply python/matplotlib.ipynb]: you are right

File: python/matplotlib.ipynb (hunk 6 above)
Comment (jpherdi) [reply python/matplotlib.ipynb]: yes, I did.

File: python/matplotlib.ipynb (hunk 1 above)
Comment (jpherdi) [reply python/matplotlib.ipynb]: fixed it.

**Review by chrbeckm — APPROVED** [review]

**Pull request merged**